src/Sarif/Autogenerated/LogicalLocation.cs

```diff
@@ -51,10 +52,12 @@ public SarifNodeKind SarifNodeKind
         public string DecoratedName { get; set; }
 
         /// <summary>
-        /// Identifies the key of the immediate parent of the construct in which the result was detected. For example, this property might point to a logical location that represents the namespace that holds a type.
+        /// Identifies the indev of the immediate parent of the construct in which the result was detected. For example, this property might point to a logical location that represents the namespace that holds a type.
```

Comment:
>indev [](start = 27, length = 5)

"indev" => "index"

Reply:
Oh, right, this comes from the schema. I made the comment there.

---
In reply to: [243719265](https://github.com/Microsoft/sarif-sdk/pull/1170#discussion_r243719265) [](ancestors = 243719265)